unity-renderer/Assets/UnityGLTF/Scripts/Editor/GLTFImporterInspector.cs

```diff
@@ -33,21 +34,26 @@ public override void OnInspectorGUI()
             EditorGUI.BeginChangeCheck();
             var importNormalsProp = serializedObject.FindProperty("_importNormals");
             var importNormals = EditorGUILayout.Popup(importNormalsProp.displayName, importNormalsProp.intValue, _importNormalsNames);
+
             if (EditorGUI.EndChangeCheck())
             {
                 importNormalsProp.intValue = importNormals;
             }
+
             EditorGUILayout.Separator();
             EditorGUILayout.LabelField("Materials", EditorStyles.boldLabel);
             EditorGUILayout.PropertyField(serializedObject.FindProperty("_importMaterials"));
             EditorGUILayout.PropertyField(serializedObject.FindProperty("_useJpgTextures"), new GUIContent("Use JPG Textures"));
 
             EditorGUILayout.Separator();
             EditorGUILayout.LabelField("Misc", EditorStyles.boldLabel);
-            EditorGUILayout.PropertyField(serializedObject.FindProperty("_importSkeleton"));
+            var importSkeleton = serializedObject.FindProperty("_importSkeleton");
+
+            if (importSkeleton != null)
+                EditorGUILayout.PropertyField(importSkeleton);
```

Comment:
The error caused by this was introduced somewhere way before the GLTF Multithread, I like this check to avoid the spamming of errors but it wasn't needed before.

Let's keep it in mind just in case something breaks in the future.